submission/persistence-filesystem/src/main/kotlin/ac/uk/ebi/biostd/persistence/filesystem/pagetab/PageTabService.kt

```diff
@@ -63,4 +66,17 @@ class PageTabService(
             size = file.size(),
             attributes = emptyList()
         )
+
+
+    private fun createTempFolder(accNo: String, version: Int): File {
```

Comment:
I noticed we already have the same logic [here](https://github.com/EBIBioStudies/biostudies-backend-services/blob/master/commons/commons-serialization-util/src/main/kotlin/uk/ac/ebi/serialization/common/FilesResolver.kt#L23). I think it might be worth it to have this in a common place, maybe something in FileUtils like `generateTempSubmissionFolder` or something like that. WDYT?

Reply:
The problem for me is that FileUtils is a class the contains IO file operations, copyOrReplaceFile, deleteFile, createHardLink, moveFile. All this operations does not have a relation with submission or even biostudies.